packages/credential-registry/schemas/ClaimedCredential.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "ClaimedCredential",
+  "description": "A [[ClaimedCredential]] represents a specific, factually-oriented claim that could be the Id and CredentialSubject of a [[VerifiableCredential]]",
+  "type": "object",
+  "properties": {
+    "did": {
+      "type": "string"
+    },
+    "type": {
+      "type": "string"
+    },
+    "typeSchema": {
+      "type": "string"
+    },
+    "value": {
+      "type": "object",
+      "properties": {
+        "keccak256": { "type": "string" },
+        "value": { "type": "string" },
+        "description": { "type": "string" },
+        "country": { "type": "string" },
+        "city": { "type": "string" },
+        "postalCode": { "type": "string" },
+        "date": { "type": "string" },
+        "time": { "type": "string" },
+        "startDate": { "type": "string" },
+        "endDate": { "type": "string" },
+        "issuanceDate": { "type": "string" },
+        "expirationDate": { "type": "string" },
+        "issuingEntity": { "type": "string" },
+        "imageIPFS": { "type": "string" }
+      }
+    },
+    "encrypted": {
+      "$ref": "#/definitions/JWE"
+    },
+    "verifiableCredentials": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/CeramicStreamId"
+      }
+    }
+  },
+  "required": ["did", "type"],
+  "additionalProperties": false,
+  "definitions": {
+    "CeramicDocId": {
```

Comment:
Looks like you updated a couple of the cases of `CeramicDocId` to `CeramicStreamId`, but there are still many other examples of `CeramicDocId` throughout this PR.  I count 15 currently.  Would be good to just replace all of those to be consistent and use modern naming conventions for Ceramic Streams

Reply:
Hi, thanks for all the comments, FYI I'm preparing a new update to the PR with all the suggestions, I'll send it in a couple of days

Reply:
Changed all ceramicDocs for ceramicStreams and CIP-82